docs/design/rubygems.md

```diff
@@ -0,0 +1,357 @@
+# WIP: Design document for RubyGems/Bundler package manager
+
+## Development prerequisites
+
+```bash
+sudo dnf install rubygems rubygems-bundler
+```
+
+## Main files
+
+```bash
+├── .bundle
+│   └── config
+├── Gemfile
+├── Gemfile.lock
+├── vendor/cache
+```
+
+To create a gem with bundler run:
+
+```bash
+bundle gem {name}
+```
+
+This will create another directory with the following structure:
+
+```bash
+├── {name}.gemspec
+├── Gemfile
+├── lib
+│   ├── {name}.rb
+├── Rakefile
+...
+```
```

Comment:
This is a bit confusing, are we talking about two different directories here? It's still just one directory right?

IMO you can just merge `{name}.gemspec` into the first file tree, I think that's all the relevant files. If you want, you can mention that one would initialize the directory with `bundle gem {name}` and the rest of the files get created with other bundler commands